### PR TITLE
Add Mobile Menu Events

### PR DESCRIPTION
**Pull request by sevinchek**
In this branch we:

- Added a new ul list for the mobile nav menu
- Updated Linter file
- Made appear and disappear menu with JS events
- Added section navigation with menu options

### CODE REVIEW

File: scripts/menu.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const mobileMenu = document.querySelector('#mob-m');
+const ulMenu = document.querySelector('.mob-nav');
+const closeMenu = document.querySelector('.close-menu');
+const menuOption1 = document.querySelector('.menu-option-1');
+const menuOption2 = document.querySelector('.menu-option-2');
+const menuOption3 = document.querySelector('.menu-option-3');
```
Comment (mricanho) [inline scripts/menu.js:6]: - [ ] You don't need 3 different classes if it's gonna do the same action, please reduce to only one class the menu-option

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @sevinchek @alonsofl, 

Good job so far! 🔥🔥 It was a really outstanding first attempt, the behavior it's the expected one and you follow the original design! Congrats!

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] Right now you only have two linters, Eslint and Stylelint, you need to add the Webhint linter on your linters.yml file, [check this link](https://github.com/microverseinc/linters-config/tree/master/html-css-js) for instructions.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

almost there

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scripts/menu.js (hunk 1 above)
Comment (sevinchek) [reply scripts/menu.js:6]: I tried to do it with only one class but didn't work.

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello again @sevinchek @alonsofl  🔥 

Sometimes happens, for this instance, you may use id. For now, your project is complete, good work!

ok ok

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sevinchek** [comment]
Thanks, @mricanho.

**Pull request merged**